### ISSUE TITLE
Hoverfly prepends a "./" to proxied request's query parameters if any parameter value contains a colon ":"

### ISSUE THREAD

**Issue by kodeninja** [SpectoLabs/hoverfly#653]
Start a simple HTTP server locally:

```javascript
const http = require('http');
const hostname = '127.0.0.1';
const port = 3000;

const server = http.createServer((req, res) => {
  res.statusCode = 200;
  res.setHeader('Content-Type', 'text/plain');
  res.end('Hello World\n');
});

server.listen(port, hostname, () => {
  console.log(`Server running at http://${hostname}:${port}/`);
});
```

Start hoverfly locally in `capture` mode, on the default port `8500`:

```bash
hoverfly -capture
```

Make a request to the server, with at least one query parameter whose value contains a colon `:`

```bash
curl -x 'http://localhost:8500' 'http://localhost:3000/hello?c=b&a=d:e'
```

Hoverfly log:

```bash
INFO[2017-09-20T13:09:52-07:00] request and response captured                 mode=capture request=&map[scheme:http destination:localhost:3000 path:/hello query:map[c:[b] a:[d:e]] headers:map[User-Agent:[curl/7.43.0] Accept:[*/*] Proxy-Connection:[Keep-Alive]] body: method:GET] response=&map[error:nil response]
```

Actual request as seen in Wireshark:

![hoverfly-request-colon-wireshark](https://user-images.githubusercontent.com/817245/30665240-31d737b0-9e05-11e7-8750-c69406e9e2f3.png)

As can be seen in the above capture, Hoverfly rearranged the query parameters, but also pre-pended a `./` to them, in effect causing the `a` parameter to become _invalid_.

This is breaking any such proxied requests that contain a colon in their query parameter values.

Is there a workaround possible for this?

**Hoverfly version**:

```bash
hoverfly --version
v0.14.0
```

**Comment by benjih** [comment]
Hey @kodeninja,

I am able to recreate this issue locally and am looking into fixing it. I anticipate a fix being in the next release of Hoverfly, v0.14.1, which I expect will be released Friday or Monday.

I do not think there is a workaround to this. 

I will update this issue when I've fixed this issue.

**Comment by kodeninja** [comment]
Thanks, @benjih, for the prompt response!

**Comment by tommysitu** [comment]
The fix had been released as v0.14.1. Please try it out.

**Comment by kodeninja** [comment]
@tommysitu, @benjih I can confirm the issue is fixed with the latest hoverfly (I tested with `v0,14,2`)!

![hoverfly-request-colon-fixed-wireshark](https://user-images.githubusercontent.com/817245/31205890-20191208-a929-11e7-9203-1e757b89c08a.png)

Thanks a lot for your help!
